### DISCUSSION TITLE
documentation:  "old" docs are gone, why?

### DISCUSSION THREAD

**Discussion by alaricljs** [zbm-dev/zfsbootmenu#557] [Questions] [answered]
### ZFSBootMenu build source

Release EFI

### ZFSBootMenu version

2.2.2

### Boot environment distribution

Arch Linux

### Problem description

I have v2.2.2 installed on my PC and having boot issues.  v2.3.0 (3 days old) is not available via my preferred install method (the aur).  In spite of the docs site supporting versioned documentation the 2.2.x docs are not available.  2.2.x only got superseded 3 days ago, it's a bit aggressive to cut documentation access so quickly on a subsystem that many people would choose not to upgrade if they are not having issues and likely prefer to fix any issue without being forced to upgrade just so the documentation they have access to matches.

### Steps to reproduce

1. go to https://docs.zfsbootmenu.org/
2. Try to change the docs version (bottom left corner)

**Comment by zdykstra** [comment, marked as answer]
I've reinstated the v2.2.x documentation section with a header indicating that it's EOL.
